### ISSUE TITLE
Add tab in description of keyboard interaction for accordion

### ISSUE THREAD

**Issue by sophieschuermans** [w3c/aria-practices#426]
The current description of keyboard interaction for accordion doesn't include the tab key. It would be useful to add it and explicitly state that each accordion header should be in the tab order.  According to the example, tab will go through all accordion headers and also through any tabbable elements of an open accordion panel.

**Comment by mcking65** [comment]
@sophieschuermans, this issue is resolved with commit 2b6dcaf.
